### DISCUSSION TITLE
How can I help with translating GitHub Docs pages?

### DISCUSSION THREAD

**Discussion by VladislavDev** [community/community#45239] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Many pages of GitHub Docs are not translated or partially translated into my native language. This is not a problem for me, but I would like to contribute and offer my own translation option for the documentation pages. Is there such a possibility?

**Comment by SilasRodrigues19** [comment, marked as answer]
You can contribute your translations to the docs repo:

https://github.com/github/docs

**Reply by VladislavDev** [reply]
Thank you very much!
I will study it. Can I ask you additional questions if they arise?
